### ISSUE TITLE
Another way to use gamemoderun

### ISSUE THREAD

**Issue by Ahmed-E-86** [FeralInteractive/gamemode#213]
In order to use gamemode I have to use `gamemoderun ./game`, or `gamemoderun wine ./game.exe`, but I want to hook another application to the game, to be more specific I want to use `firejail`, and I can't use both, because if I use `firejail gamemoderun wine ./game.exe`, `firejail` wont work, and if I use `gamemoderun firejail wine ./game.exe`, `gamemoderun` wont work, so I wonder if is there another way to use gamemode, or that is it?

**Comment by stephanlachnit** [comment]
> I wonder if is there another way to use gamemode, or is that it?

Yes there is, you can hook into the process if you know the PID, see `gamemoded --help`.

**Comment by Ahmed-E-86** [comment]
Do you mean the PID of the executable of the game, or the PID of wineserver?
